infrastructure/template.yaml

```diff
@@ -61,15 +79,23 @@ Resources:
           Value: !Ref 'Environment'
         - Key: Source
           Value: "https://github.com/alphagov/di-account-management-backend/blob/main/infrastructre/template.yaml"
+      PointInTimeRecoverySpecification:
+              PointInTimeRecoveryEnabled: true
```

Comment:
Can I just double check the indentation is right on this one @divyaparameswaran ?

I know YAML can be a bit of a pain about indents, this looks like it's a few below line 82?